tests/conftest.py

```diff
@@ -0,0 +1,133 @@
+import argparse
+from dataclasses import dataclass
+from functools import cached_property
+from importlib import import_module
+from sys import version_info as _version_info
+from types import ModuleType
+from typing import List, Type, Union
+
+import pytest
+
+C_EXT_MARK = pytest.mark.c_extension
+PY_38_AND_BELOW = _version_info < (3, 9)
```

Comment:
this compat var is probably no longer necessary

Reply:
I was waiting until 3.8 moved to unsupported on https://devguide.python.org/versions/

Reply:
Oh, right. I just figured since you were removing it from other projects, you'd want that here too.